### ISSUE TITLE
Code toggle for non config files

### ISSUE THREAD

**Issue by davidsneighbour** [gohugoio/hugoDocs#1299]
I am trying to find out, how the codetoggle shortcode works. I can find it only in connection to file names (like codetoggle filename config) and then seems to use TOML inside and the other formats seem to be produced automatically. 

What I want to do is for instance have a three formats codetoggle on this section: https://gohugo.io/content-management/menus/#simple

The toml way of adding a menu was requested and inside of frontmatter there are single brackets around `[menu.main]` which is nowhere documented. 

Long story short: Is it possible to add codetoggles without having to add a filename? And is it true that either the content of the codetoggle is TOML or one of TOML, YAML, JSON and will be transformed into the other formats AUTOMATICALLY or did I miss something?

I somehow remember having seen documentation for that shortcode, but couldn't find it and from the usage it's not clear to me how it works. I might go on a PR spree to add more togglers to older samples.

**Comment by jmooring** [comment]
> Is it possible to add codetoggles without having to add a filename?

```text
{{< code-toggle >}}
menu: ["main", "footer"]
{{< /code-toggle >}}
```

> is it true that either the content of the codetoggle is TOML or one of TOML, YAML, JSON

Yes, any of those.

> and will be transformed into the other formats AUTOMATICALLY

Yes.
